asmcomp/loongarch64/arch.mli

```diff
@@ -0,0 +1,76 @@
+# 2 "asmcomp/loongarch64/arch.mli"
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                yala <zhaojunchao@loongson.cn>                          *)
+(*                                                                        *)
+(*               Copyright © 2008-2023 LOONGSON                           *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(* Specific operations for the LoongArch processor *)
```

Comment:
Please decide which spelling, `LoongArch` or `Loongarch`, you want to use, and be consistent across this PR. `arch.ml` in this directory uses `Loongarch`, other files seem to prefer `LoongArch`.